### PR TITLE
DASH-920: Added license

### PR DESCRIPTION
**Pull request by Nnamdi101**
### Proposed Changes

*Replace this text with an overview of what changed and why.*

### Checklist

*Place an `x` inside the brackets to check off items.*

- [x] I have [updated the change log](https://github.com/virtru/dev-guide/blob/master/process/merge-and-tag.md#change-log)
- [x] I have updated the `VERSION` file if merging to `master`
- [ ] I have added tests to cover any new functionality or bugfixes

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```
Comment (svirtryu) [inline LICENSE.md]: 🤔

Comment (Nnamdi101) [reply LICENSE.md]: Lol, the PR wasn't ready yet.

File: auditexport/auditclient/auditclient.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
 API_PATH = '/api/messages'
 CLIENT_NAME = 'AuditPythonClient:v2.0.0'
 API_VERSION = 'v2'
+IAT_LEEWAY_SECONDS = 300.0
```
Comment (svirtryu) [inline auditexport/auditclient/auditclient.py]: Why do we need to put the leeway here?

Comment (Nnamdi101) [reply auditexport/auditclient/auditclient.py]: Same reason as the canaries. The clock on the server may be running slower than the machine that runs this code

**Review by svirtryu — APPROVED** [review]

**Pull request merged**